### ISSUE TITLE
Document what happens exactly when `forever stop` is called

### ISSUE THREAD

**Issue by fresheneesz** [foreverjs/forever#802]
I'm hoping that when `forever stop` is called, it sends a `SIGTERM` signal to let the process gracefully exit, but looking at the code, I'm not seeing anything that verifies it. And in any case, the documentation should mention this. I also see some "forceKill" option or something in the code, which makes me think there are other things going on. Can someone clarify this?

**Comment by egirshov** [comment]
Indeed, the documentation is very brief on that, but check the `killSignal` option (iirc, you need to specify it in `forever start` so that the stop will behave accordingly).

**Comment by AlexanderElias** [comment]
I think I found the the issue to this. I created a new issue: [https://github.com/foreverjs/forever/issues/811](url)

**Comment by fresheneesz** [comment]
Your link is messed up for some reason: https://github.com/foreverjs/forever/issues/811

**Comment by AlexanderElias** [comment]
@fresheneesz Thanks I believe it it fixed now.
